equinix/resource_fabric_service_profile_acc_test.go

```diff
@@ -17,139 +17,135 @@ import (
 	v4 "github.com/equinix-labs/fabric-go/fabric/v4"
 )
 
-func testAccFabricReadServiceProfileConfig(uuid string) string {
-	return fmt.Sprintf(`data "equinix_fabric_service_profile" "test" {
-	uuid = "%s"
-	}
-`, uuid)
-}
+func TestAccFabricCreateServiceProfile_PFCR(t *testing.T) {
+	ports := GetFabricEnvPorts(t)
 
-func testAccFabricReadServiceProfilesListConfig(name string) string {
-	return fmt.Sprintf(`data "equinix_fabric_service_profiles" "test" {
-		filter {
-			property = "/name"
-			operator = "="
-			values = ["%s"]
-		}
+	var portTcFirst, portTcSecond v4.Port
```

Comment:
Same as first comment:

We shouldn't use structs for the values that will be passed inside of resource.ParallelTest. They need to be primitives. If we use more than one layer deep of the v4.Port struct type, and the ENV variable is not set then it will fail the test run when TF_ACC is not set.

There's a GHA workflow that runs non-terraform tests and it will cause this workflow to fail.

Please use the primitive type matching whichever subcategory of `v4.Port` you'll be using.

Reply:
done